wiki/User_title/en.md

```diff
@@ -24,9 +26,22 @@ The colour of a user title is the same as the user's colour, which is also usual
 
 <!-- the bottom post on https://web.archive.org/web/20131107133826/https://osu.ppy.sh/forum/t/141240 -->
 
-On the forum sections of the [old osu! website](https://old.ppy.sh)<!-- TODO: wiki page -->, users could obtain titles by reaching certain post count milestones. These were only shown on the sidebars of forum posts, not the users' profiles, and were not shown if a custom title was already set.
-
-<!-- TODO: list all of the titles and their requirements -->
+On the forum sections of the [old osu! website](https://old.ppy.sh)<!-- TODO: wiki page -->, users could obtain titles by reaching certain post count milestones as follows: <!-- https://osu.ppy.sh/community/forums/posts/2306316 -->
+
+| Forum title | Total post count |
+| :-: | :-- |
+| Rhythm Rookie | 0-4 |
+| Tempo Trainee | 5-14 |
+| Whistle Blower | 15-29 |
+| Cymbal Sounder | 30-49 |
+| Beat Clicker | 50-79 |
+| Slider Savant | 80-119 |
+| Spinner Sage | 120-179 |
+| Star Shooter | 180-259 |
+| Combo Commander | 260-499 |
+| Rhythm Incarnate | 500+ |
+
+These titles were only visible on the sidebars of forum posts (not on the users' profiles) and were not shown if another custom title was already set prior.
```

Comment:
hyphen -> en dash
```suggestion
On the forum sections of the [old osu! website](https://old.ppy.sh)<!-- TODO: wiki page -->, users could obtain titles by reaching certain post count milestones: <!-- https://osu.ppy.sh/community/forums/posts/2306316 -->

| Forum title | Total post count |
| :-: | :-- |
| Rhythm Rookie | 0–4 |
| Tempo Trainee | 5–14 |
| Whistle Blower | 15–29 |
| Cymbal Sounder | 30–49 |
| Beat Clicker | 50–79 |
| Slider Savant | 80–119 |
| Spinner Sage | 120–179 |
| Star Shooter | 180–259 |
| Combo Commander | 260–499 |
| Rhythm Incarnate | 500+ |

These titles were only visible on the sidebars of forum posts (not on the users' profiles) and were not shown if another custom title was already set prior.
```
or if you prefer the "as follows:"
```suggestion
On the forum sections of the [old osu! website](https://old.ppy.sh)<!-- TODO: wiki page -->, users could obtain titles by reaching certain post count milestones as follows: <!-- https://osu.ppy.sh/community/forums/posts/2306316 -->

| Forum title | Total post count |
| :-: | :-- |
| Rhythm Rookie | 0–4 |
| Tempo Trainee | 5–14 |
| Whistle Blower | 15–29 |
| Cymbal Sounder | 30–49 |
| Beat Clicker | 50–79 |
| Slider Savant | 80–119 |
| Spinner Sage | 120–179 |
| Star Shooter | 180–259 |
| Combo Commander | 260–499 |
| Rhythm Incarnate | 500+ |

These titles were only visible on the sidebars of forum posts (not on the users' profiles) and were not shown if another custom title was already set prior.
```

Reply:
yup